domain/src/DomainValidator.php

```diff
@@ -104,7 +105,7 @@ public function validate(DomainInterface $domain) {
       }
     }
     // Allow modules to alter this behavior.
-    \Drupal::moduleHandler()->invokeAll('domain_validate', $error_list, $hostname);
+    \Drupal::moduleHandler()->invokeAll('domain_validate', array($error_list, $hostname));
```

Comment:
Good catch. We do not have a test for this yet. (note to self: would need to go in domain_test.module)

There is more to this though, `domain.api.php` defines an alter hook (`hook_domain_validate_alter()`) where as the like above is a hook. So it should probably be changed to `\Drupal::moduleHandler()->alter('domain_validate', $error_list, $hostname, $context);`

Which one do you prefer?